### DISCUSSION TITLE
Two issues.

### DISCUSSION THREAD

**Discussion by chrcoluk** [f1multiviewer/issue-tracker#517] [General]
Since is no bug report section will post this here.

1 - I had to drop to CPU rendering for the app, its still enabled for video decoding.  Otherwise the app was very laggy and streams ran at around 5 fps (not measured but extremely low FPS), hardware is RTX 3080 so should be more than enough GPU power.
2 - App keeps prompting me to update but website has not been kept up to date stuck on 1.31.7 for a while.  I am using portable version.

**Comment by JJWatMyself** [comment]
For the first issue you can disable vsync. We have a few nvidia settings covereded on the website for nvidia.

**Comment by chrcoluk** [comment]
Thanks, I will try disabling vsync to see if it fixes the GPU problem.

It has worked great, thank you.

**Comment by LapsTimeOFF** [comment]
If you are still having those issues, please open an issue in the issues tab or contact us on [Discord](https://muvi.gg/discord)
